### PR TITLE
Updating properties table and removing get/set

### PR DESCRIPTION
**Pull request by digitaljoshua**
Remove get/set
Update properties table

### CODE REVIEW

**Review by GiselleLeeOSI — APPROVED** [review]
Looks good to me.

File: Content_Portal/Documentation/Identity/Identity_ClientCredentialClient.md [hunk 1]
```diff
@@ -18,14 +18,14 @@ Client Credential clients are used for machine-to-machine communication without
...
-Name | string | Gets or sets name of Client.
-Enabled | bool | Gets or sets whether client is enabled. Client can be used for authentication if set to true. Client cannot be used for authentication if set to false.
-AccessTokenLifetime | int32 | Gets or sets lifetime of access token issued for this client after authentication. Minimum 60 seconds. Maximum 3600 seconds. Defaults to 3600 seconds.
-Tags | string[] | Gets or sets for OSIsoft internal use only.
+RoleIds | Guid[] | List of Roles to be assigned to this client. Member role is always required. For security reasons we advise against assigning Admin roles to a client.
+Id | string | Secret Id.
+Name | string | Name of Client.
+Enabled | bool | Whether client is enabled. Client can be used for authentication if set to true. Client cannot be used for authentication if set to false.
+AccessTokenLifetime | int32 | Lifetime of access token issued for this client after authentication. Minimum 60 seconds. Maximum 3600 seconds. Defaults to 3600 seconds.
+Tags | string[] | Fof or OSIsoft internal use only.
```
Comment (GiselleLeeOSI) [inline Content_Portal/Documentation/Identity/Identity_ClientCredentialClient.md:28]: Typo in line 28 "Fof or"

File: Content_Portal/Documentation/Management/Account_ClientRole_1.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ Client `Roles` authorize API requests made by clients to various OCS features. C
 
 For HTTP requests and responses, the ClientRole object has the following properties and JSON-serialized body: 
 
-| Property | Type | Description | 
+| Property Name | Data Type | Description | 
  | --- | --- | ---  | 
 | ClientId | string | Client Id. | 
 | RoleId | string | Role Id. |
```
Comment (GiselleLeeOSI) [inline Content_Portal/Documentation/Management/Account_ClientRole_1.md:20]: In line 19 and 20, the description repeats the property name itself. Also there is punctuation here that is absent elsewhere.

File: Content_Portal/Documentation/Management/Account_UserRole_1.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ but all users have the Account Member role. The following are currently availabl
 
 For HTTP requests and responses, the UserRole object has the following properties and JSON-serialized body: 
 
-| Property | Type | Description | 
+| Property Name | Data Type | Description | 
  | --- | --- | ---  | 
 | UserId | string | User Id. |
```
Comment (GiselleLeeOSI) [inline Content_Portal/Documentation/Management/Account_UserRole_1.md:19]: Same for lines 19 and 20. Also, I think we decided to use ID in the descriptor (see glossary terms).

File: Content_Portal/Documentation/Identity/Identity_ClientCredentialClient.md (hunk 1 above)
Comment (digitaljoshua) [reply Content_Portal/Documentation/Identity/Identity_ClientCredentialClient.md]: fixed

File: Content_Portal/Documentation/Management/Account_UserRole_1.md (hunk 3 above)
Comment (digitaljoshua) [reply Content_Portal/Documentation/Management/Account_UserRole_1.md:19]: modified the descriptions

File: Content_Portal/Documentation/Management/Account_ClientRole_1.md (hunk 2 above)
Comment (digitaljoshua) [reply Content_Portal/Documentation/Management/Account_ClientRole_1.md]: Modified the descriptions

**Pull request merged**